custom_components/alexa_media/helpers.py

```
@@ -31,47 +31,86 @@
+        else:
             _LOGGER.debug(
-                "%s: Unable to add devices: %s",
+                "%s: Unable to add %d device(s): %s",
                 account,
-                EXCEPTION_TEMPLATE.format(type(ex).__name__, ex.args),
+                len(devices),
+                message,
             )
-    else:
-        return True
+    except BaseException as ex:  # pylint: disable=broad-except
```

Ruff (0.14.11)

Do not catch blind exception: `BaseException`

(BLE001)